generic/tests/cfg/pvpanic.cfg

```diff
@@ -0,0 +1,45 @@
+- pvpanic: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = driver_in_use
+    run_bgstress = pvpanic
+    disable_shutdown = yes
+    monitor_type = qmp
+    event_check = "GUEST_PANICKED"
+    guest_status = "guest-panicked"
+    driver_name = "pvpanic"
+    timeout = 360
+    RHEL.6:
+        kdump_enable_cmd = "chkconfig kdump on && service kdump restart"
+        kdump_disable_cmd = "chkconfig kdump off && service kdump stop"
+    RHEL.7:
+        kdump_enable_cmd = "chkconfig kdump on && systemctl restart kdump.service"
+        kdump_disable_cmd = "chkconfig kdump off && systemctl stop kdump.service"
+    nmi_enable_cmd = "echo 1 > /proc/sys/kernel/unknown_nmi_panic"
+    nmi_disable_cmd = "echo 0 > /proc/sys/kernel/unknown_nmi_panic"
+    NMICrashDump_enable_cmd = 'wmic class stdregprov call SetDwordValue hDefKey="&h80000002" sSubKeyName="SYSTEM\CurrentControlSet\Control\CrashControl" sValueName="NMICrashDump" uValue=1'
+    NMICrashDump_disable_cmd = 'wmic class stdregprov call DeleteValue hDefKey="&h80000002" sSubKeyName="SYSTEM\CurrentControlSet\Control\CrashControl" sValueName="NMICrashDump"'
+    Windows:
+        no Host_RHEL.m6
```

Comment:
why not "no Host_RHEL.m5"?

Reply:
When I modified patch1098,  I asked xuhan, he said, avocado is only supported on rhel6 and rhel7, so no need "no Host_RHEL.m5" .